src/applications/mhv-landing-page/routes.jsx

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import PageNotFound from '@department-of-veterans-affairs/platform-site-wide/PageNotFound';
+import { MhvSecondaryNav } from '@department-of-veterans-affairs/mhv/exports';
 import App from './containers/App';
 
 const routes = (
-  <Switch>
-    <Route exact path="/" key="mhvLandingPage">
-      <App />
-    </Route>
-    <Route>
-      <PageNotFound />
-    </Route>
-  </Switch>
+  <>
+    <MhvSecondaryNav />
+    <Switch>
+      <Route exact path="/" key="mhvLandingPage">
+        <App />
+      </Route>
+      <Route>
+        <PageNotFound />
+      </Route>
+    </Switch>
+  </>
```

Comment:
Do we want the secondary nav to show when the App component renders a Unregistered message, or a Unverified message? Does this component need to be a child of the App component?

Reply:
We don't want it to show. I wanted it a different PR just to get this initial version in.

Reply:
Cool. This is fine since we're feature toggling the component.